joplin/templates/joplin_UI/application_banner.html

```diff
@@ -16,7 +16,7 @@
         <div class="avatar circle coa-avatar"><img src="{% avatar_url user %}" alt="avatar" /></div>
         <div class="user user-account-dropdown coa-user-name" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
           {{ request.user.first_name }} {{ request.user.last_name }}
-          <img class="coa-user-dropdown-arrow" src="/static/images/icon_down_arrow.svg" alt="Down Arrow Icon">
+          <img class="coa-user-dropdown-arrow" src="{% static 'images/icon_down_arrow.svg' %}" alt="Down Arrow Icon">
```

Comment:
This is hopefully the problem with why the arrow was 404ed on staging. I didn't add the handler for enviroment vars...